### PR TITLE
adding template changing example

### PR DESCRIPTION
**Pull request by tuopouk**
I added the template changing app that you enhanced.

### CODE REVIEW

File: examples/template-change.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    "xgridoff",
+    "ygridoff",
+    "gridon",
+    "none",
+]
+
+change_figure_template = html.Div(
+    [
+        html.Div("Change Figure Template"),
+        dcc.Dropdown(figure_templates, figure_templates[0], id="template"),
```
Comment (Coding-with-Adam) [inline examples/template-change.py:32]:
```suggestion
        dcc.Dropdown(options=figure_templates, value=figure_templates[0], id="template"),
```

File: examples/template-change.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+
```
Comment (Coding-with-Adam) [inline examples/template-change.py:3]: @tuopouk 
Thank you for submitting this app. It's a very good app for new Dash community member to learn how to update their graph templates. 
 
Can you please update all the IDs in this file to be unique IDs. For example, the 'template' id could be `template-change-x-template`. 
You can read more about [unique IDs here](https://github.com/AnnMarieW/dash-app-gallery/blob/main/CONTRIBUTING.md#unique-ids).

**Comment by Coding-with-Adam** [comment]
Thank you @tuopouk. I think the template-change app is ready. 

In regard to the countdown app, can you please add unique IDs here as well, and remove all list brackets around Callback outputs, inputs and states. In older versions of Dash, that was necessary, but not anymore. 

@AnnMarieW, in the countdown app, do you think we can implement the audio part of the app by adding the mp3 to the assets folder, or would that not work here?

**Comment by AnnMarieW** [comment]
>do you think we can implement the audio part of the app by adding the mp3 to the assets folder, or would that not work here?
Yes, I think it should work - let's give it a try.

**Pull request merged**